services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/grade_level_filters.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+
+import { arrayFromNumbers, } from './shared'
+
+import NumberSuffixBuilder from '../../../../components/modules/numberSuffixBuilder'
+import { TwoThumbSlider, Tooltip, helpIcon, } from '../../../../../Shared/index'
+
+interface GradeLevelFiltersProps {
+  gradeLevelFilters: number[],
+  handleGradeLevelFilterChange: (gradeLevelFilters: number[]) => void,
+}
+
+const GRADE_LEVEL_LABELS = ['2-3', '4-5', '6-7', '8-9', '10-12']
```

Comment:
(In reference to my earlier comment) I think if we don't have these constantized elsewhere, it would be nice to have them defined since we do use them in multiple places

Reply:
Yeah, so the readability grade bands (which are in fact the same as these, but only incidentally, which is to say they could be different so we don't necessarily want to share a constant), are defined as a constant in the shared file. The grade level labels we use here are not quite the same as the ones we use for display in the activity row itself (there are spaces around the hyphen and we omit 2-3), so we can't use the same constant for those. Since the labels aren't actually shared and since in both places they are used for display purposes, not business logic, I actually think it makes sense to not abstract them further than necessary. Let me know if you feel strongly though and we can talk about it more.